app/templates/main/faq.html

```diff
@@ -0,0 +1,34 @@
+{% extends 'layouts/base.html' %}
+
+{% block content %}
+    <div class="ui text container">
+        <h1>FAQ</h1>
+
+        <h3>Is it true that idling your engine saves more money than turning it off and back on?</h3>
+        <p>No, in fact, if you idle for longer than 10 seconds you are wasting more gas. It used to be true, with older vehicles that had carburetors, that turning an engine on would consume substantial amounts of fuel. However, modern vehicles (anything past the mid-80s) use fuel-injection and this uses almost no fuel to start an engine.</p><br>
+
+        <h3>Is idling a vehicle the best way to get it ready to drive ("warmed-up")?</h3>
+        <p>No, idling causes unnecessary wear and tear on your vehicle. The best way, by far, to get the engine warmed up is to drive it. Modern engines respond best when driven, not sitting idling.</p><br>
+
+        <h3>What about getting the interior warm or cold, is idling ok for that?</h3>
+        <p>No, driving the vehicle is the quickest way to get warm or cold air into the interior of the vehicle.</p><br>
+
+        <h3>Does idling really waste that much gas?</h3>
+        <p>Yes, idling a vehicle can waste between 1/5 - 7/10 of a gallon of gas per hour.</p><br>
```

Comment:
Would percentages have better effect here, or do you think that fractions are the most communicative?